teams/tidb/membership.json

```diff
@@ -90,6 +90,7 @@
     "reviewers": [
         "ChenPeng2013",
         "Defined2014",
+	"fixdb",
```

Comment:
more spaces to align with the previous lines?

Reply:
Oh, i used the tab wrongly